brainglobe_atlasapi/atlas_generation/atlas_scripts/kim_mouse_20um.py

```diff
@@ -0,0 +1,257 @@
+                "id": ROOT_ID,
+                "name": "root",
+                "acronym": "root",
+                "structure_id_path": [ROOT_ID],
+                "rgb_triplet": [255, 255, 255]
+            }
+
+        Returns
+        -------
+            list of dictionaries containing the atlas information.
+        """
+        if ONTOLOGY_FILE not in self.download_dir_content:
+            raise Exception(
+                f"{ONTOLOGY_FILE} not found in {self.download_folder}"
+            )
+        df = pd.read_csv(self.download_folder / ONTOLOGY_FILE)
+        int_cols = ["id", "RGB_1", "RGB_2", "RGB_3", "structure_id_path"]
+        df[int_cols] = df[int_cols].apply(pd.to_numeric, errors="coerce")
+        df = df.dropna(subset=["id"])
+        df[int_cols] = df[int_cols].astype("int32")
+        structures = [
+            {
+                "id": ROOT_ID,
+                "name": "root",
+                "acronym": "root",
+                "structure_id_path": [ROOT_ID],
+                "rgb_triplet": [255, 255, 255],
+            }
+        ]
+        id_to_row = df.set_index("id").to_dict("index")
+        for _, row in df.iterrows():
+            structures.append(
+                {
+                    "id": row["id"],
+                    "name": row["name"],
+                    "acronym": row["acronym"],
+                    "structure_id_path": AtlasBuilder.get_path_to_root_id(
+                        id_to_row, df, row["id"]
+                    ),
+                    "rgb_triplet": [row["RGB_1"], row["RGB_2"], row["RGB_3"]],
+                }
+            )
+        self.structures = structures
+        return self.structures
+
+    def retrieve_or_construct_meshes(self) -> Dict[int, str]:
+        """
+        This function should return a dictionary of ids and corresponding paths to
+        mesh files. Some atlases are packaged with mesh files, in these cases we
+        should use these files. Then this function should download those meshes.
+        In other cases we need to construct the meshes ourselves. For this we have
+        helper functions to achieve this.
+        """
+        meshes_dir = self.download_folder
+        meshes_dict = construct_meshes_from_annotation(
+            meshes_dir, self.annotation, self.structures, ROOT_ID
+        )
+        return meshes_dict
+
+    @staticmethod
+    def get_path_to_root_id(
+        id_to_row: Dict, df: pd.DataFrame, current_id: int
+    ) -> List:
+        path_to_root = []
+        while True:
+            current_row = id_to_row.get(current_id)
+            if current_row is None:
+                break
+            path_to_root.insert(0, current_id)
+            parent_id = current_row["structure_id_path"]
+            if parent_id == current_id:
+                break
+            current_id = parent_id
+
+        path_to_root.insert(0, ROOT_ID)
+        return path_to_root
+
+    @staticmethod
+    def load_as_asr(path: Path) -> np.ndarray:
```

Comment:
Can I suggest setting `ORIENTATION` to the orientation of the original raw array data (ignoring header metadata if present)?
Then, I think this transformation can be removed, because the `wrapup_atlas_from_data` function will take care of transforming to ASR.
This would reduce the thinking needed when we come back to this in the future :)